myapp/app/controllers/tasks_controller.rb

```diff
@@ -2,10 +2,16 @@
 
 class TasksController < ApplicationController # rubocop:todo Style/Documentation
   def index
-    @tasks = Task.order(created_at: :asc)
-    return unless params[:sort] == 'latest'
-
-    @tasks = Task.order(created_at: :desc)
+    @tasks = case params[:sort]
+             when 'latest'
+               Task.order(created_at: :desc)
+             when 'closest-deadline'
+               Task.order(due_date: :asc)
+             when 'longest-deadline'
+               Task.order(due_date: :desc)
+             else
+               Task.order(created_at: :asc)
+             end
```

Comment:
You don't have to fix it for now, but I think this approach is likely to become difficult in the future 👀  Because if you add other conditions, you will have to change all `Task.order` each time.
You may want to try creating a method only where you create a sort, for example! please consider this in future steps 👍

Reply:
Yep, actually i was thinking about making it portable function and changed the way in the step 13 though, but thanks for the catch and suggestion.